qtpandas/views/CSVDialogs.py

```diff
@@ -120,7 +120,7 @@ def _initUI(self):
         layout.addWidget(self.otherSeparatorLineEdit)
         self.setLayout(layout)
 
-    @Slot('QBool')
+    @Slot('bool')
```

Comment:
I meant bool as in the python type rather than a string that says 'bool' - sorry lol..